packages/react/src/Card/CardHeading.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+import { Heading, HeadingProps } from '../Heading'
+
+export type CardHeadingProps = Partial<HeadingProps> & PropsWithChildren<HTMLAttributes<HTMLHeadingElement>>
+
+export const CardHeading = forwardRef(
+  (
+    { children, className, level = 1, size = 'level-4', ...restProps }: CardHeadingProps,
```

Comment:
Do we want to set a default level here? We don't really know where Card will be in the document structure I think.